addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1189,6 +1190,20 @@ var SnippetsMenu = Widget.extend({
         }
         return functions;
     },
+    /**
+     * Install a module
+     *
+     * @private
+     * @param {number} moduleId: the module to install
+     * @return {Deferred}
+     */
+    _installModule: function (moduleId) {
```

Comment:
@qsm-odoo No clue where to place this method it in file.

Reply:
In the private methods of SnippetsMenu is ok but put it at the correct place (alphabetical order...)